screwdriver/screwdriver.go

```diff
@@ -0,0 +1,176 @@
+package screwdriver
```

Comment:
Most code from this file is from https://github.com/screwdriver-cd/launcher/blob/master/screwdriver/screwdriver.go